README.md

```diff
@@ -1,16 +1,5 @@
+# iron-scroll-target-behavior
 
-<!---
-
-This README is automatically generated from the comments in these files:
-
-
-Edit those files, and our readme bot will duplicate them over here!
-Edit this file, and the bot will squash your changes :)
```

Comment:
yeah, I rebased against master to be able to merge, so that's why the edit. It should be overridden by tedium once merged into master though.